File: parent/pinyougou-web-user/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){

    //分页查询订单集合
    this.search=function(searchMap){
        return $http.post('order/search.do',searchMap);
    }

    //取消订单
    this.cancleOrder=function(orderId){
        return $http.get('../order/cancleOrder.do?orderId='+orderId);
    }
});

File: parent/pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface OrderService {
    void add(Order order);

    Map<String, Object> search(Map<String, String> searchMap);

    void cancleOrder(Long orderId);
	PageResult search(Integer page, Integer rows, Order order);

    List creatPic();
}
